creator-node/src/serviceRegistry.js

```diff
@@ -245,6 +246,8 @@ class ServiceRegistry {
                   value.length > 100
                     ? `[Truncated array with ${value.length} elements]`
                     : this._truncateBull(value, newDepth)
+              } else if (_.isEmpty(value)) {
+                json[key] = value
```

Comment:
I'm pretty sure we want it to be set to null because otherwise I think it would just not show the key/pair at all, so that wouldn't be an accurate reflection of what data the jobs are inputting and outputting